include/intx/int128.hpp

```diff
@@ -542,16 +542,22 @@ inline constexpr unsigned clz(uint128 x) noexcept
 }
 
 
-inline uint64_t bswap(uint64_t x) noexcept
+inline constexpr uint64_t bswap(uint64_t x) noexcept
 {
-#ifdef _MSC_VER
-    return _byteswap_uint64(x);
-#else
+#if __has_builtin(__builtin_bswap64)
```

Comment:
Which compilers support this?

Reply:
GCC and clang

Reply:
I mean versions :wink:

Reply:
You mean `__has_builtin` or `__builtin_bswap64`?

Reply:
Both. On what compilers will that codepath be enabled? As that is our preferred codepath I reckon.